sdk/python/feast/repo_operations.py

```diff
@@ -133,6 +138,17 @@ def parse_repo(repo_root: Path) -> RepoContents:
                     # Don't add if the push source's batch source is a duplicate of an existing batch source
                     if push_source_dep not in data_sources_set:
                         res.data_sources.append(push_source_dep)
+            elif isinstance(obj, StreamFeatureView):
```

Comment:
I think you also want to add a `and not any((obj is sfv) for sfv in res.stream_feature_views)` check here?

the reason this check exists above is if to handle the case where you define a FV in one file and import it in another file - if you don't deduplicate by `is` equality, the FV will get applied twice

Reply:
fixed.